release-notes.txt

```diff
@@ -1,3 +1,528 @@
++ boards/nucleo-l4r5zi : add PWM configuration (#20036)
++ boards/nucleo-l552ze: add PWM configuration (#20032)
++ boards/p-nucleo-wb55: add PWM configuration (#20120)
++ boards: add support for Adafruit Feather nRF52840 Sense (#20027)
++ drivers/include/periph: add FREQM peripheral driver (#20020)
++ drivers/ws281x: Add gpio_ll and timer based driver (#19891)
++ pyterm: add native support (#20172)
++ sys/shell: add firmware version to version cmd (#20244)
+* AVR-8: Implement Power Management (#19784)
+* boards/nrf52dk: Minimal Arduino pinout support (#20286)
+* boards/same54-xpro: configure pins for UART HW flow control &
+  ethernet LED (#20183)
+* boards: remove `extern mtd_dev_t *mtd<n>` declarations from board
+  definitions (#20104)
+* makefiles/tests/tests.inc.mk: use native term for tests (#20215)
+* treewise: typo "or RAM" (#20219)
+
+CPU
+---
+
+* cpu/msp430: rework MSP430 x1xx periph drivers (#19835)
+* cpu/nrf5x: clean up periph_uart (#20102)
+* cpu/stm32: FMC used for low-level LCD parallel interface (#19943)
++ cpu/esp32/periph: add low-level SDMMC peripheral driver for ESP32 and
+  ESP32-S3 (#19786)
++ cpu/msp430: improve periph_timer (#20160)
++ cpu/stm32/periph: add low-level SDMMC peripheral driver (#19540)
++ cpu/atmega_common: implement periph_timer_query_freqs (#20142)
++ cpu/cc26xx_cc13xx: implement periph_timer_query_freqs (#20143)
++ cpu/kinetis: implement periph_timer_query_freqs (#20144)
++ cpu/nrf5x_common: implement periph_timer_query_freqs (#20145)
++ cpu/qn908x: implement periph_timer_query_freqs (#20146)
++ cpu/sam0_common: implement periph_timer_query_freqs (#20147)
++ cpu/stm32: add ADCAL operation (#20044)
++ cpu/stm32: implement periph_timer_query_freqs (#20148)
++ drivers/periph/uart: add periph_uart_tx_ondemand feature (#20108)
+* cpu/avr8_common: allow to build with !periph_pm (#20247)
+* cpu/avr8_common: fix build with !core_thread (#20240)
+* cpu/nrf5x_common: clean up UART implementation (#20111)
+* cpu/sam0/uart: implement uart_pin_cts()/uart_pin_rts() (#20171)
+* cpu/sam0_common: implement periph_gpio_ll and periph_gpio_ll_irq (#20189)
+* cpu/stm32/periph_spi: only perform DMA transfer above threshold (#20214)
+* cpu/stm32: uart: don't do DMA for small transfers (#20230)
+
+Device Drivers
+--------------
+
++ driver/mtd: mtd_init add documentation for return value (#20033)
++ drivers/periph_timer: add timer_query_freqs() (#16349)
+* drivers/mtd_flashpage: improve _write_page (#20173)
+* drivers/mtd: improve doc consistency (#20184)
+* drivers: make use of busy_wait() (#20258)
+
+Documentation
+-------------
+
+* boards/nucleo-l433rc: doc update - image and MCU table (#20098)
+* boards/nucleo-l452re: doc update (#20207)
+* boards/p-nucleo-wb55: documentation update (#20115)
+* doc/mainpage: don't claim to be a microkernel (#20246)
+* doc: Extend and take responsibility for 802.15.4 security roadmap (#20123)
+* doc: Move roadmap in from the wiki (#20122)
```

Comment:
```suggestion
* doc: Move and update roadmap in from the wiki (#20122, #20123)
```

The latter justifies why it says "update" there, but standalone the title only makes sense in a PR context where the author is visible, but has no business as a standalone entry in change logs.